### PR TITLE
[FIX] web: remove bottom border of first table raw

### PR DESCRIPTION
**Pull request by adwid**
When using the boxed layout, if the business document has a table, its
first row (after the table head) will have a bottom border and the last
cell in the row has a white background.

To reproduce the error:
(Need timesheet_grid. Use demo data)
1. In Settings > General Settings, Change Document Template:
    - Select the second template
2. In Timesheets, select list view
3. Select several lines
4. Print > Timesheet Entries

Error: In PDF, the first timesheet row has a bottom line and its last
cell has a white background (instead of grey).

The bottom line should only be applied on row in table header. The
background color should also be applied on first row in table body.

OPW-2478311

### CODE REVIEW

**Comment by adwid** [comment]
@nle-odoo

**Review by nle-odoo — APPROVED** [review]
I'm not sure what `compat 12.0` mean, since we are in 12 it doesn't make sense to be compatible with 12.0 so maybe this mean "make 12.0 compatible with what was available before".

In 11.0 the style was:

```scss
.o_report_layout_boxed {
    // This is not a general rule since the id is not used in all the reports,
    // we should make a general structure for report body to use it everywhere
    #informations {
        border: 1px solid @gray;
        padding-top: 10px;
        margin: 0;
    }
    > h2 {
        text-transform: uppercase;
    }
    table {
        border: 1px solid @gray;
        thead {
            tr th {
                border-bottom: 3px solid @gray !important;
                text-transform: uppercase;
            }
        }
        tbody {
            color: @gray;
            tr {
                td {
                    border-top: none;
                    border-right: 1px solid @gray;
                }
                td:last-child {
                    background-color: @gray-lighter;
                }
            }
        }
    }
    /*Total table*/
    .row > .col-xs-4 > table {
        td {
            border-right: none;
        }
        tr:last-child td, tr:first-child td {
            /*we could get rid of this rule if .border-black td would be terminated in report.less*/
            border-top: none !important;
        }
        tr:first-child td {
            background-color: transparent;
        }
        tr:last-child td {
            background-color: @gray;
            color: #fff;
        }
    }
}
````

that mostly target thead / tbody so the change seems ok to me (and this doesn't seem to cause any change in picking, delivery slip, invoice, sale order reports).

robodoo delegate+

**Comment by adwid** [comment]
robodoo r+
